funnel/models/project.py

```diff
@@ -191,6 +192,13 @@ class Project(UuidMixin, BaseScopedNameMixin, Model):
         read={'all'},
         datasets={'primary', 'without_parent', 'related'},
     )
+
+    thumbnail_image = with_roles(
```

Comment:
This is technically a preview or a cover image, not a thumbnail. We need a more appropriate name.